roles/setup/README.md

```diff
@@ -212,6 +212,69 @@ name, and values a dict of the rest of the key-value pairs of the work
 definition.  See <https://receptor.readthedocs.io/en/latest/k8s.html>
 for more information.
 
+---
+
+    local_receptor: false
+
+Put it to true if you want to use a local receptor bin (or download it from github) instead of package release for your distro.
```

Comment:
```suggestion
Set to true to upload a local receptor binary instead of installing via a package manager. If no local binary is present, it will be downloaded from receptor Releases on github.
```